tests/databases/test_bigquery.py

```diff
@@ -99,7 +99,11 @@ def test_bigquery_create_table_with_columns(database_table_fixture):
     """Test table creation with columns data"""
     database, table = database_table_fixture
 
-    statement = f"SELECT * FROM {table.metadata.schema}.INFORMATION_SCHEMA.COLUMNS WHERE table_name='{table.name}'"
+    # Looking for specific columns in INFORMATION_SCHEMA.COLUMNS as Bigquery can add/remove columns in the table.
+    statement = (
+        f"SELECT TABLE_CATALOG, TABLE_SCHEMA, TABLE_NAME, COLUMN_NAME, DATA_TYPE FROM {table.metadata.schema}"
```

Comment:
What are those extra columns btw? This looks good to be but it could be replaced with the values on this assert: https://github.com/astronomer/astro-sdk/runs/7830499547?check_suite_focus=true#step:11:765

Reply:
@sunank200  `column_default` is the col added. Which is not listed in https://cloud.google.com/bigquery/docs/information-schema-columns